src/components/Instructors.js

```diff
@@ -25,125 +25,104 @@ const useStyles = makeStyles((theme) => ({
+          <MailOutline />
+        </ListItemIcon>
+        <ListItemText secondary='N/A' />
+      </ListItem>
+      <ListItem>
+        <ListItemIcon>
+          <Domain />
+        </ListItemIcon>
+        <ListItemText secondary='N/A' />
+      </ListItem>
+    </List>
+  )
+}
+
+const Instructor = ({ classes, instructors }) => {
+  return instructors.map((instructor, i) => {
+    return (
+      <List
+        subheader={
+          <ListSubheader tabIndex='0' className={classes.instructor} disableSticky={true}>
+            {`${instructor.firstName} ${instructor.lastName}`}
+          </ListSubheader>
+        }
+        key={i}
+      >
+        <ListItem>
+          <ListItemIcon>
+            <MailOutline />
+          </ListItemIcon>
+          <ListItemText
+            secondary={
+              instructor.email !== 'N/A' ? (
+                <a href={'mailto:' + instructor.email} rel='noopener noreferrer' target='_blank'>
+                  {instructor.email}
+                </a>
+              ) : (
                 'N/A'
-              }
-            />
-          </ListItem>
-          <ListItem>
-            <ListItemIcon>
-              <Domain />
-            </ListItemIcon>
-            <ListItemText
-              secondary={
-                (!Object.is(teachers.office, 'N/A') && (
-                  <a
-                    href={getMapUrl(teachers[i].office, true)}
-                    rel='noopener noreferrer'
-                    target='_blank'
-                  >
-                    {teachers[i].office}
-                  </a>
-                )) ||
+              )
+            }
+          />
+        </ListItem>
+        <ListItem>
+          <ListItemIcon>
+            <Domain />
+          </ListItemIcon>
+          <ListItemText
+            secondary={
+              instructor.office !== 'N/A' ? (
+                <a
+                  href={getMapUrl(instructor.office, true)}
+                  rel='noopener noreferrer'
+                  target='_blank'
+                >
+                  {instructor.office}
+                </a>
+              ) : (
                 'N/A'
-              }
-            />
-          </ListItem>
-        </List>
-      )
-    }
-  }
-  return list
+              )
+            }
+          />
+        </ListItem>
+      </List>
+    )
+  })
 }
 
-const Instructors = (props) => {
-  const { teachers } = props
+export default function Instructors(props) {
+  const { instructors } = props
   const [open, setOpen] = useState(false)
   const classes = useStyles()
 
   return (
     <div>
-      <Button
-        onClick={() => setOpen(true)}
-        color='secondary'
-        className={classes.text}
-      >
-        {teachers.length >= 2 && 'Instructors'}
-        {teachers.length < 2 && 'Instructor'}
+      <Button onClick={() => setOpen(true)} color='secondary' className={classes.text}>
+        {instructors.length <= 1 ? 'Instructor' : 'Instructors'}
```

Comment:
Should this check for equal to 1?

Reply:
I thought about this one too. I think my reasoning was that if there's no instructor, the dialog displays the information for one instructor, it's just all `N/A`. So it made more sense to have `Instructor` vs `Instructors`. I don't know if that makes sense